### PR TITLE
*: add release notes for 1.0 RC.1

### PR DESCRIPTION
**Pull request by csuzhangxc**
### What problem does this PR solve? 

- add release notes for 1.0 RC.1

### What is changed and how it works?

- add release notes for 1.0 RC.1

### Check List 

Tests 

 - No code

### CODE REVIEW

**Comment by csuzhangxc** [comment]
@GregoryIan @ericsyh PTAL

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,29 @@
 
 All notable changes to this project will be documented in this file.
 
+## [1.0-rc.1] 2019-07-01
+
+- Remove the restriction of "the next DDL statement cannot be executed unless the current DDL operation is completely finished" for incremental migration [#177](https://github.com/pingcap/dm/pull/177)
```
Comment (GregoryIan) [inline CHANGELOG.md]:
```suggestion
- Remove the restriction of "the next shard DDL statement cannot be executed unless the current shard DDL operation is completely finished in shard merge scene for incremental migration" [#177](https://github.com/pingcap/dm/pull/177)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,29 @@
...
+- Support auto retry when meeting the error of `invalid connection` for incremental migration [#66](https://github.com/pingcap/dm/pull/66)
```
Comment (GregoryIan) [inline CHANGELOG.md]:
```suggestion
- Support retry task on the `invalid connection` error for binlog replication [#66](https://github.com/pingcap/dm/pull/66)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,29 @@
...
+- Support generating `-schema-create.sql` files automatically for full migration [#186](https://github.com/pingcap/dm/pull/186)
+- Use [TiDB SQL Parser](https://github.com/pingcap/parser) to parse and restore DDL statements in binlog query events, and remove the `using` statement when replicating to downstream [#54](https://github.com/pingcap/dm/pull/54)
```
Comment (GregoryIan) [inline CHANGELOG.md]:
```suggestion
- Use [TiDB SQL Parser](https://github.com/pingcap/parser) to parse and restore DDL statements in binlog query events, and remove the `using db` statement when replicating to downstream [#54](https://github.com/pingcap/dm/pull/54)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,29 @@
...
+- Support migrate tables with generated column for full and incremental migration [#42](https://github.com/pingcap/dm/pull/43) [#60](https://github.com/pingcap/dm/pull/60)
```
Comment (GregoryIan) [inline CHANGELOG.md]:
```suggestion
- Support migrate tables with generated column for full migration and binlog replication [#42](https://github.com/pingcap/dm/pull/43) [#60](https://github.com/pingcap/dm/pull/60)
```

**Comment by GregoryIan** [comment]
Rest LGTM

File: CHANGELOG.md [hunk 5]
```diff
@@ -2,6 +2,29 @@
...
+- Redirect the log of mydumper into the log of DM-worker [#93](https://github.com/pingcap/dm/pull/93)
+- Support using `--print-sample-config` command flag to show the sample config of DM-worker and DM-master [#28](https://github.com/pingcap/dm/pull/28)
+- Support showing the status of sharding DDL replication in the Grafana dashboard and optimize the dashboard [#96](https://github.com/pingcap/dm/pull/96) [#101](https://github.com/pingcap/dm/pull/101) [#120](https://github.com/pingcap/dm/pull/120)
+- Support setting the `max_allowed_packet` parameter for the database connection to the upstream and downstream [#99](https://github.com/pingcap/dm/pull/99) 
+- Support using `unsafe_cleanup` in DM Ansible to clean components of DM cluster [#128](https://github.com/pingcap/dm/pull/128)
+- Fix the wrong flush for checkpoint when existing multiple sharding groups [#124](https://github.com/pingcap/dm/pull/124)
+- Fix the wrong progress when loading dumped files [#89](https://github.com/pingcap/dm/pull/89)
+- Improve the test framework and test cases greatly and improve the correctness significantly
+- Fix multiple other bugs that don't affect correctness
```
Comment (GregoryIan) [inline CHANGELOG.md]:
```suggestion
- Fix many other bugs that don't affect correctness
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -2,6 +2,29 @@
...
+- Redirect the log of mydumper into the log of DM-worker [#93](https://github.com/pingcap/dm/pull/93)
+- Support using `--print-sample-config` command flag to show the sample config of DM-worker and DM-master [#28](https://github.com/pingcap/dm/pull/28)
+- Support showing the status of sharding DDL replication in the Grafana dashboard and optimize the dashboard [#96](https://github.com/pingcap/dm/pull/96) [#101](https://github.com/pingcap/dm/pull/101) [#120](https://github.com/pingcap/dm/pull/120)
+- Support setting the `max_allowed_packet` parameter for the database connection to the upstream and downstream [#99](https://github.com/pingcap/dm/pull/99) 
+- Support using `unsafe_cleanup` in DM Ansible to clean components of DM cluster [#128](https://github.com/pingcap/dm/pull/128)
+- Fix the wrong flush for checkpoint when existing multiple sharding groups [#124](https://github.com/pingcap/dm/pull/124)
+- Fix the wrong progress when loading dumped files [#89](https://github.com/pingcap/dm/pull/89)
+- Improve the test framework and test cases greatly and improve the correctness significantly
```
Comment (GregoryIan) [inline CHANGELOG.md]:
```suggestion
- Greatly improve the test to ensure correctness
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -2,6 +2,29 @@
...
+- Optimize RCP framework to limit the concurrency between DM-master and DM-worker [#157](https://github.com/pingcap/dm/pull/157)
+- Redirect the log of mydumper into the log of DM-worker [#93](https://github.com/pingcap/dm/pull/93)
+- Support using `--print-sample-config` command flag to show the sample config of DM-worker and DM-master [#28](https://github.com/pingcap/dm/pull/28)
+- Support showing the status of sharding DDL replication in the Grafana dashboard and optimize the dashboard [#96](https://github.com/pingcap/dm/pull/96) [#101](https://github.com/pingcap/dm/pull/101) [#120](https://github.com/pingcap/dm/pull/120)
+- Support setting the `max_allowed_packet` parameter for the database connection to the upstream and downstream [#99](https://github.com/pingcap/dm/pull/99) 
+- Support using `unsafe_cleanup` in DM Ansible to clean components of DM cluster [#128](https://github.com/pingcap/dm/pull/128)
+- Fix the wrong flush for checkpoint when existing multiple sharding groups [#124](https://github.com/pingcap/dm/pull/124)
+- Fix the wrong progress when loading dumped files [#89](https://github.com/pingcap/dm/pull/89)
```
Comment (GregoryIan) [inline CHANGELOG.md]:
```suggestion
- Fix the wrong progress status when loading dumped files [#89](https://github.com/pingcap/dm/pull/89)
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -2,6 +2,29 @@
...
+- Optimize RCP framework to limit the concurrency between DM-master and DM-worker [#157](https://github.com/pingcap/dm/pull/157)
+- Redirect the log of mydumper into the log of DM-worker [#93](https://github.com/pingcap/dm/pull/93)
+- Support using `--print-sample-config` command flag to show the sample config of DM-worker and DM-master [#28](https://github.com/pingcap/dm/pull/28)
+- Support showing the status of sharding DDL replication in the Grafana dashboard and optimize the dashboard [#96](https://github.com/pingcap/dm/pull/96) [#101](https://github.com/pingcap/dm/pull/101) [#120](https://github.com/pingcap/dm/pull/120)
+- Support setting the `max_allowed_packet` parameter for the database connection to the upstream and downstream [#99](https://github.com/pingcap/dm/pull/99) 
+- Support using `unsafe_cleanup` in DM Ansible to clean components of DM cluster [#128](https://github.com/pingcap/dm/pull/128)
+- Fix the wrong flush for checkpoint when existing multiple sharding groups [#124](https://github.com/pingcap/dm/pull/124)
```
Comment (GregoryIan) [inline CHANGELOG.md]:
```suggestion
- Fix flush checkpoint wrongly when existing multiple sharding groups [#124](https://github.com/pingcap/dm/pull/124)
```

File: CHANGELOG.md [hunk 9]
```diff
@@ -2,6 +2,29 @@
...
+- Support migrate tables with generated column for full and incremental migration [#42](https://github.com/pingcap/dm/pull/43) [#60](https://github.com/pingcap/dm/pull/60)
+- Support appending the task name as a suffix to the dumped data directory (`dir` in the task configuration file) for full migration [#100](https://github.com/pingcap/dm/pull/100)
+- Support resuming tasks automatically after the DM-worker process restarted [#88](https://github.com/pingcap/dm/pull/88) [#116](https://github.com/pingcap/dm/pull/116)
+- Support skipping pre-check items when starting tasks [#65](https://github.com/pingcap/dm/pull/65)
+- Support case-insensitive binlog event type in Binlog Event Filter rules [#188](https://github.com/pingcap/dm/pull/188)
+- Optimize relay log mechanism to improve compatibility with upstream MySQL/MariaDB [#92](https://github.com/pingcap/dm/pull/92) [#108](https://github.com/pingcap/dm/pull/108) [#117](https://github.com/pingcap/dm/pull/117) [#140](https://github.com/pingcap/dm/pull/140) [#171](https://github.com/pingcap/dm/pull/171)
+- Optimize RCP framework to limit the concurrency between DM-master and DM-worker [#157](https://github.com/pingcap/dm/pull/157)
```
Comment (GregoryIan) [inline CHANGELOG.md]:
```suggestion
- Optimize RPC framework to limit the concurrency between DM-master and DM-worker [#157](https://github.com/pingcap/dm/pull/157)
```

File: CHANGELOG.md (hunk 4 above)
Comment (ericsyh) [inline CHANGELOG.md]:
```suggestion
- Support migrating tables with generated column for full and incremental migration [#42](https://github.com/pingcap/dm/pull/43) [#60](https://github.com/pingcap/dm/pull/60)
```

File: CHANGELOG.md [hunk 10]
```diff
@@ -2,6 +2,29 @@
...
+- Support generating `-schema-create.sql` files automatically for full migration [#186](https://github.com/pingcap/dm/pull/186)
+- Use [TiDB SQL Parser](https://github.com/pingcap/parser) to parse and restore DDL statements in binlog query events, and remove the `using` statement when replicating to downstream [#54](https://github.com/pingcap/dm/pull/54)
+- Support migrate tables with generated column for full and incremental migration [#42](https://github.com/pingcap/dm/pull/43) [#60](https://github.com/pingcap/dm/pull/60)
+- Support appending the task name as a suffix to the dumped data directory (`dir` in the task configuration file) for full migration [#100](https://github.com/pingcap/dm/pull/100)
+- Support resuming tasks automatically after the DM-worker process restarted [#88](https://github.com/pingcap/dm/pull/88) [#116](https://github.com/pingcap/dm/pull/116)
+- Support skipping pre-check items when starting tasks [#65](https://github.com/pingcap/dm/pull/65)
+- Support case-insensitive binlog event type in Binlog Event Filter rules [#188](https://github.com/pingcap/dm/pull/188)
```
Comment (ericsyh) [inline CHANGELOG.md]:
```suggestion
- Support case-insensitive in binlog event type of Binlog Event Filter rules [#188](https://github.com/pingcap/dm/pull/188)
```

**Comment by csuzhangxc** [comment]
@GregoryIan @ericsyh PTAL again

File: CHANGELOG.md [hunk 11]
```diff
@@ -2,6 +2,29 @@
 
 All notable changes to this project will be documented in this file.
 
+## [1.0-rc.1] 2019-07-01
+
+- Remove the restriction of "the next shard DDL statement cannot be executed unless the current shard DDL operation is completely finished in shard merge scene for binlog replication" [#177](https://github.com/pingcap/dm/pull/177)
+- Support retry task on the `invalid connection` error for binlog replication [#66](https://github.com/pingcap/dm/pull/66)
+- Support generating `-schema-create.sql` files automatically for full migration [#186](https://github.com/pingcap/dm/pull/186)
+- Use [TiDB SQL Parser](https://github.com/pingcap/parser) to parse and restore DDL statements in binlog query events, and remove the `using db` statement when replicating to downstream [#54](https://github.com/pingcap/dm/pull/54)
+- Support migrating tables with generated column for full migration and binlog replication [#42](https://github.com/pingcap/dm/pull/43) [#60](https://github.com/pingcap/dm/pull/60)
```
Comment (kennytm) [inline CHANGELOG.md]: Is this PR 42 or 43?

Comment (csuzhangxc) [reply CHANGELOG.md]: Good catch! fixed in https://github.com/pingcap/dm/pull/192/commits/8c8bdefc0adf06922abe7495e5156f847a9da50e.

**Comment by ericsyh** [comment]
LGTM

**Comment by GregoryIan** [comment]
lgtm

**Pull request merged**